<!DOCTYPE html>
<html>
<head>
  <title>On the importance of small changes in RNA expression.</title>
  <meta charset="utf-8">
  <meta name="description" content="On the importance of small changes in RNA expression.">
  <meta name="author" content="Keith Hughitt">
  <meta name="generator" content="slidify" />
  <meta name="apple-mobile-web-app-capable" content="yes">
  <meta http-equiv="X-UA-Compatible" content="chrome=1">
  <link rel="stylesheet" href="libraries/frameworks/io2012/css/default.css" media="all" >
  <link rel="stylesheet" href="libraries/frameworks/io2012/phone.css" 
    media="only screen and (max-device-width: 480px)" >
  <link rel="stylesheet" href="libraries/frameworks/io2012/css/slidify.css" >
  <link rel="stylesheet" href="libraries/highlighters/highlight.js/css/solarized_light.css" />
  <base target="_blank"> <!-- This amazingness opens all links in a new tab. -->
  <script data-main="libraries/frameworks/io2012/js/slides" 
    src="libraries/frameworks/io2012/js/require-1.0.8.min.js">
  </script>
  
    <link rel="stylesheet" href = "assets/css/ribbons.css">

</head>
<body style="opacity: 0">
  <slides class="layout-widescreen">
    
    <!-- LOGO SLIDE -->
    <!-- END LOGO SLIDE -->
    

    <!-- TITLE SLIDE -->
    <!-- Should I move this to a Local Layout File? -->
    <slide class="title-slide segue nobackground">
      <hgroup class="auto-fadein">
        <h1>On the importance of small changes in RNA expression.</h1>
        <h2>St Laurent, G., Shtokalo, D., Tackett, M.R., Yang, Z., Vyatkin, Y. Milos, P.M., Seilheimer, B., McCaffrey, T.A., Kapranov, P.</h2>
        <p>Keith Hughitt<br/></p>
      </hgroup>
      <a href="https://github.com/khughitt/slidify-small-changes-in-rna-expression/zipball/gh-pages" class="example">
        Download
      </a>
          </slide>

    <!-- SLIDES -->
      <slide class="segue dark" id="slide-1" style="background:;">
  <hgroup>
    
  </hgroup>
  <article>
    <!-- Custom Styles -->

<style type='text/css'>
    slides > slide {
        height: 800px;
        margin-top: -400px;
    }
    img {
        max-height: 560px;
        max-width: 964px;
    }
    slide a {border-bottom: none;}
    .references li { font-size: 18px; }
</style>

<h2>Background</h2>

  </article>
  <!-- Presenter Notes -->
</slide>

      <slide class="" id="slide-2" style="background:;">
  <hgroup>
    <h2>Motivation</h2>
  </hgroup>
  <article>
    <ul class = "build">
<li>Often transcript profiling (microarray, RNA-Seq, etc) experiment are
conducted, and &quot;differentially expressed&quot; genes are determined based on 
thresholds for:

<ol>
<li>Fold change (FC)</li>
<li>p-value</li>
</ol></li>
<li>Setting a minimum p-value makes sense, but what about the FC requirement?
There are several possible reasons for setting a minumum fold change.</li>
<li>There are at least a couple reasons this is done:

<ol>
<li>Noisy data collection (ligation, amplification, imperfect sampling)</li>
<li>Assumption that most interested biological effects will involve large
increases/decreases in transcript levels.</li>
</ol></li>
<li>However:

<ul>
<li>Are these valid assumptions?</li>
<li>What effect do these thresholds have on the loss of biological 
information?</li>
</ul></li>
</ul>

  </article>
  <!-- Presenter Notes -->
</slide>

      <slide class="" id="slide-3" style="background:;">
  <hgroup>
    <h2>Approach</h2>
  </hgroup>
  <article>
    <p>Figure 1: Example fold changes for markers of early immune response</p>

<p><img src="assets/img/1-s2.0-S1046202313000960-gr1.jpg" alt="Fig1"></p>

  </article>
  <!-- Presenter Notes -->
</slide>

      <slide class="segue dark" id="slide-4" style="background:;">
  <hgroup>
    <h2>Results</h2>
  </hgroup>
  <article>
    
  </article>
  <!-- Presenter Notes -->
</slide>

      <slide class="" id="slide-5" style="background:;">
  <hgroup>
    <h2>Question: How often are p-value and/or FC cutoffs applied?</h2>
  </hgroup>
  <article>
    <h3>Approach</h3>

<ul class = "build">
<li>Pulled 40 most recent papers in PubMed for query &quot;microarray expression
profiling&quot; (...It would be interesting to know results for &quot;RNA-Seq&quot;)</li>
</ul>

<h3>Result</h3>

<ul class = "build">
<li>At least 50% of studied applied a FC cutoff (the rest used p-values only)</li>
<li>This means that any biologically relevant DE genes below the FC cutoff
will be missed.</li>
</ul>

  </article>
  <!-- Presenter Notes -->
</slide>

      <slide class="" id="slide-6" style="background:;">
  <hgroup>
    <h2>Question: Which statistical methods work best for detecting low FC genes?</h2>
  </hgroup>
  <article>
    <h3>Approach</h3>

<ol class = "build">
<li>Compared performance of six different methods for detecting genes with &lt; 2
FC:

<ul>
<li>edgeR</li>
<li>t-test</li>
<li>DESeq</li>
<li>baySeq</li>
<li>Welch&#39;s test</li>
<li>NOIseq</li>
</ul></li>
<li><span class='blue'>Performance metric</span>: Enrichment of a subset of
the 23 inflammation-related GO categories.

<ul>
<li>For each method, sorted genes by p-value, selected top 2000 genes, and
selected upregulated genes with FC &lt; 2.</li>
<li>GO enrichment evaluated using the Fisher exact test, adjusted for
multiple testing with the Benjamini-Hochberg correction.</li>
<li>Control: randomized timepoints</li>
</ul></li>
</ol>

  </article>
  <!-- Presenter Notes -->
</slide>

      <slide class="" id="slide-7" style="background:;">
  <hgroup>
    <h2>Question: Which statistical methods work best for detecting low FC genes?</h2>
  </hgroup>
  <article>
    <h3>Result</h3>

<ol class = "build">
<li>Overall, methods all performed pretty similarly:

<ul>
<li><span class='blue'>Number genes</span>

<ul>
<li>355 (DESeq)</li>
<li>410 (edgeR)</li>
</ul></li>
<li><span class='blue'>p-value (Immune system response)</span>

<ul>
<li>4.36E-41 (baySeq)</li>
<li>3.91E-47 (t-test)</li>
</ul></li>
</ul></li>
<li>Best method, however: Keep any gene found by at least one of the methods

<ul>
<li>631 genes</li>
<li>p-value: &lt;1E-50</li>
</ul></li>
<li><span class='red'>Conclusion</span>: Existing methods are sufficient to
detect biologically meaningful enrichment at low FC.</li>
</ol>

  </article>
  <!-- Presenter Notes -->
</slide>

      <slide class="" id="slide-8" style="background:;">
  <hgroup>
    <h2>References</h2>
  </hgroup>
  <article>
    <!-- Custom JavaScript -->

<script src="http://ajax.aspnetcdn.com/ajax/jQuery/jquery-1.7.min.js"></script>

<script type='text/javascript'>
$(function() {
    $("p:has(img)").addClass('centered');
});
</script>

  </article>
  <!-- Presenter Notes -->
</slide>

    <slide class="backdrop"></slide>
  </slides>

  <!--[if IE]>
    <script 
      src="http://ajax.googleapis.com/ajax/libs/chrome-frame/1/CFInstall.min.js">  
    </script>
    <script>CFInstall.check({mode: 'overlay'});</script>
  <![endif]-->
</body>
<!-- Grab CDN jQuery, fall back to local if offline -->
<script src="http://ajax.aspnetcdn.com/ajax/jQuery/jquery-1.7.min.js"></script>
<script>window.jQuery || document.write('<script src="libraries/widgets/quiz/js/jquery-1.7.min.js"><\/script>')</script>
<!-- Load Javascripts for Widgets -->
<!-- MathJax: Fall back to local if CDN offline but local image fonts are not supported (saves >100MB) -->
<script type="text/x-mathjax-config">
  MathJax.Hub.Config({
    tex2jax: {
      inlineMath: [['$','$'], ['\\(','\\)']],
      processEscapes: true
    }
  });
</script>
<script type="text/javascript" src="http://cdn.mathjax.org/mathjax/2.0-latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>
<!-- <script src="https://c328740.ssl.cf1.rackcdn.com/mathjax/2.0-latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML">
</script> -->
<script>window.MathJax || document.write('<script type="text/x-mathjax-config">MathJax.Hub.Config({"HTML-CSS":{imageFont:null}});<\/script><script src="libraries/widgets/mathjax/MathJax.js?config=TeX-AMS-MML_HTMLorMML"><\/script>')
</script>
<!-- LOAD HIGHLIGHTER JS FILES -->
<script src="libraries/highlighters/highlight.js/highlight.pack.js"></script>
<script>hljs.initHighlightingOnLoad();</script>
<!-- DONE LOADING HIGHLIGHTER JS FILES -->
</html>